src/pages/groupFeed/GroupFeed.tsx

```diff
@@ -1,5 +1,62 @@
+import styled from '@emotion/styled';
+
+import CuriousArticle from './CuriousArticle';
+import CuriousProfile from './CuriousProfile';
+import GroupCuriousTitle from './GroupCuriousTitle';
+import GroupSideHeader from './GroupSideHeader';
+import GroupTodayWriteStyle from './GroupTodayWriteStyle';
+import Spacing from '../../components/commons/Spacing';
+
 const GroupFeed = () => {
-  return <div>GroupFeed</div>;
+  return (
+    <GroupFeedWrapper>
+      <GroupFeedThumnail></GroupFeedThumnail>
```

Comment:
p2 ) 중간에 넣는 거 없으면 self-closing 태그로 바꿔주세용 ~
```suggestion
      <GroupFeedThumnail />```